src/travis2docker/templates/.vscode/settings.json

```diff
@@ -1,3 +1,14 @@
 {
-  "python.analysis.extraPaths": ["/home/odoo/instance/odoo"]
+  "editor.rulers": [120],
+  "files.autoSave": "afterDelay",
+  "files.autoSaveDelay": 500,
+  "python.analysis.extraPaths": ["/home/odoo/instance/odoo"],
+  "python.linting.enabled": true,
+  "pylint.args": [
+    "--load-plugins", "pylint-odoo",
+    "--rcfile", "${env:PRE_COMMIT_HOME}/cfg/.pylintrc"
+  ],
+  "flake8.args": [
+    "--config", "${env:PRE_COMMIT_HOME}/cfg/.flake8"
+  ],
```

Comment:
Still in the process of enabling it since I can't call `eslint` from the containers t2d is currently creating, the extension needs `eslint` to be available in order to lint.

Reply:
Just as an FYI @moylop260, I added support for eslint, however I had to upgrade node which means manually installing a newer version so `eslint` works.